src/ReactiveUI.Splat.Tests/ReactiveUI.Splat.Tests.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.0" />
+    <PackageReference Include="Splat.Autofac" Version="7.*" />
+    <PackageReference Include="Splat.DryIoc" Version="7.*" />
+    <PackageReference Include="Splat.Ninject" Version="7.*" />
+    <PackageReference Include="xunit" Version="2.4.0" />
```

Comment:
Remove these, they are old and the directory.build.props files will get these.

Reply:
The xunit is what i'm referring to btw.

Reply:
removed.